backends/ebpf/tests/ptf/test.py

```diff
@@ -587,3 +587,57 @@ def runTest(self):
         pkt[IP].dst = 0x11993355  # mask is 0xFF00FFFF
         testutils.send_packet(self, PORT0, pkt)
         testutils.verify_packet(self, pkt, PORT1)
+
+
+class LPMTableCachePSATest(P4EbpfTest):
+    p4_file_path = "p4testdata/table-cache-lpm.p4"
+    p4c_additional_args = "--table-caching"
+
+    def runTest(self):
+        # TODO: make this additional entry working
+        # self.table_add(table="ingress_tbl_lpm", key=["00:11:22:33:44:50/44"], action=1, data=["11:22:33:44:55:67"])
+        self.table_add(table="ingress_tbl_lpm", key=["00:11:22:33:44:55/48"], action=1, data=["11:22:33:44:55:66"])
+        pkt = testutils.simple_ip_packet(eth_dst="00:11:22:33:44:50")
+        exp_pkt = testutils.simple_ip_packet(eth_dst="11:22:33:44:55:66")
+        exp_pkt[Ether].type = 0x8601
+
+        testutils.send_packet(self, PORT0, pkt)
+        testutils.verify_packet(self, exp_pkt, PORT1)
+
+        # modify cache directly - verify that it is used (modify to NoAction)
+        self.table_update(table="ingress_tbl_lpm_cache",
+                          key=["32w0x60", "32w0", "64w0x5044332211000000"], action=0, data=["160w0"])
+        testutils.send_packet(self, PORT0, pkt)
+        testutils.verify_packet(self, pkt, PORT1)
+
+        # update table - verify clearing cache
+        self.table_update(table="ingress_tbl_lpm", key=["00:11:22:33:44:55"], action=1, data=["11:22:33:44:55:66"])
```

Comment:
Same here, can we do get on the table cache and see if there are no entries?

Reply:
Another reason would be to test `nikss-ctl table update` and `nikss-ctl table del` which are nowhere else executed.

Reply:
ok let's keep as it is but add a more informative comment